Makefile

```
@@ -46,3 +43,42 @@ release: dist/lisk-core-v$(CORE_VERSION)
 dist/lisk-core-v$(CORE_VERSION):
 	npx oclif-dev pack --targets=linux-x64,darwin-x64
 endif
+
+build: build-image build-local
+
+build-image:
+	docker build -f ./docker/Dockerfile --build-arg NODEJS_VERSION=$(shell cat .nvmrc) --tag=lisk/core .
```

Should we add version while tagging image?
```suggestion
	docker build -f ./docker/Dockerfile --build-arg NODEJS_VERSION=$(shell cat .nvmrc) --tag=lisk/core:$(CORE_VERSION) .
```

---

No, this is mostly for the development purpose. The exact version tagging is done and uploaded to the DockerHub by the DevOps post release.